tutorials/models_mapping/plot_1_tensor_models.py

```diff
@@ -0,0 +1,333 @@
+
+
+"""
+Tensor Meshes
+=============
+
+Here we demonstrate various ways that models can be defined and mapped to
+tensor meshes. Some things we consider are:
+
+    - Surface topography
+    - Adding structures of various shape to the model
+    - Parameterized models
+    - Models with 2 or more physical properties
+    
+
+"""
+
+#########################################################################
+# Import modules
+# --------------
+#
+
+from SimPEG import Mesh
+from SimPEG import Utils
+from SimPEG import Maps
+import numpy as np
+import matplotlib.pyplot as plt
+
+# sphinx_gallery_thumbnail_number = 3
+
+#############################################
+# Defining the mesh
+# -----------------
+#
+# Here, we create the tensor mesh that will be used for all examples.
+#
+
+
+def makeExampleMesh():
```

Comment:
might be worth making `dh`, `npad`, `ncore` inputs to the function so that if the use wants to play around a bit it is easy for them to change the mesh size. 

Also, to be more in-line with [python community practices](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names), lets use lowercase and underscores for names, e.g. `make_example_mesh` 
```suggestion
def make_example_mesh(dh=5, npad=5, ncore=20):
```

Reply:
sure, ill change this